### PR TITLE
beaglebone black tutorial

### PR DESCRIPTION
**Pull request by cshields**
Added a beaglebone black tutorial for readthedocs  (also slipped in a couple of minor fixes to the raspi2 tutorial that I caught while reusing most of its text)

### CODE REVIEW

**Comment by cshields** [comment]
@johngian r?

File: client/bbb-install.rst [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+I like to manage SatNOGS with supervisord. There are plenty of other ways to do it and your mileage may vary. Before we get to the SatNOGS client we have one a dependency that has not yet been discussed: rotctld for providing rotor control interface to the SatNOGS Arduino board. This should have been installed with libhamlib-utils above.
+
+As with SatNOGS, I run rotctld through supervisord. Open your favorite editor and drop this into
+/etc/supervisord/conf.d/rotctld.conf::
+
+   [program:rotctld]
+   command=/usr/bin/rotctld -m 202 -r /dev/ttyACM0 -s 19200 -T 127.0.0.1
+   autostart=true
+   autorestart=true
+   user=pi
```
Comment (johngian) [inline client/bbb-install.rst]: This doesn't look right. I assume you copy/pasted it from raspi2 docs. It should be ``debian``.

File: client/bbb-install.rst [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+   directory=/home/debian/
+   autostart=true
+   autorestart=true
+   user=debian
+   environment=SATNOGS_SQLITE_URL="sqlite:////tmp/jobs.sqlite",SATNOGS_API_URL="https://network.satnogs.org/api/",SATNOGS_API_TOKEN="<TOKEN>",SATNOGS_VERIFY_SSL="TRUE",SATNOGS_STATION_ID="<ID>",SATNOGS_STATION_LAT="<LATITUDE>",SATNOGS_STATION_LON="<LONGITUDE>",SATNOGS_STATION_ELEV="<ELEVATION>",SATNOGS_PPM_ERROR="<PPM>"
+
+Obviously there are fields above that will need configured appropriately, your latitude/longitude/elevation (example: 43.210 -86.123, elevation is in meters), API token, station ID, and PPM. Log in to the SatNOGS Network console and click on your user icon in the upper-right, then "My Profile". If you have not already added your ground station to the web site please do so now with the "Add Ground Station" button. Once that is done your ground station ID will be shown. In this screen as well you can click the "API Key" button for the token needed in the configuration above. All settings that can be changed in the environment can be found in the [settings.py file](https://github.com/satnogs/satnogs-client/blob/master/satnogsclient/settings.py)
```
Comment (johngian) [inline client/bbb-install.rst]: ``/s/need configured/need to be configured/g`` ?

**Comment by johngian** [comment]
r+wc :rocket:

File: client/bbb-install.rst (hunk 1 above)
Comment (cshields) [inline client/bbb-install.rst]: good catch, thanks!

**Pull request merged**